modules/virt-switching-displays.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-mediated-devices.adoc
+
+:_content-type: PROCEDURE
+[id="virt-switching-displays_{context}"]
+= Switching between VM displays
+
+If your virtual machine (VM) has multiple displays, you can view each one by switching displays in the web console.
+
+.Prerequisites
+
+* The mediated device is configured in the `HyperConverged` custom resource and assigned to a VM.
+
+.Procedure
+
+. In the OpenShift Virtualization console, click *Workloads* -> *Virtualization* from the side menu.
+. Click the *Virtual Machines* tab.
+. Select a Windows virtual machine to open the *Virtual Machine Overview* screen.
+. Click the *Console* tab.
+. Select the appropriate display from the *Send Key* drop down menu or by using the tabs below the drop down menu.
```

Comment:
Can we add more information about what key combination to send for each display ?
```suggestion
. Select the appropriate display from the *Send Key* drop down menu or by using the tabs below the drop down menu, send key combination Alt+Ctl+1 for the first display and Alt+Ctl+2 for the second displat.
```